app/imports/client/react/actions/containers/ActionAddContainer.js

```diff
@@ -48,8 +49,15 @@ const ActionAddContainer = ({
       organizationId,
       isOpen,
       toggle,
-      linkedTo: linkedToField,
-      initialValues: getActionFormInitialState(user),
+      linkedTo,
+      initialValues: {
+        active: 0,
+        owner: getUserOptions(user),
+        toBeCompletedBy: getUserOptions(user),
+        planInPlace: ActionPlanOptions.NO,
+        // TODO: Update based on linked documents like creation modal?
+        completionTargetDate: moment().add(1, 'days'),
```

Comment:
dates should be plain js objects
```suggestion
        completionTargetDate: moment().add(1, 'days').toDate();
```